library/CMakeLists.txt

```diff
@@ -67,31 +67,40 @@ else()
 	message("-- Building ${Green}dynamic${ColourReset} library.")
 endif()
 
-
+if (CONAN_EXPORTED)
+	message("-- INFO: ${Green}Conan detected${ColourReset}... finding packages...")
+	find_package(OpenSSL REQUIRED COMPONENTS SSL Crypto)
+	find_package(ZLIB REQUIRED)
+	find_package(Opus)
+	find_package(Threads REQUIRED)
+	find_package(Git QUIET)
```

Comment:
Git and Threads are not managed by Conan, shouldn't that be "common" and independent of CONAN_EXPORTED?

Reply:
they are only here because if we run the build through conan, we wanted to group the find_package together for easy reference, it makes no difference if they are here or elsewhere apart from ease of development for us